f/ansible-navigator.json

```
@@ -32,12 +32,25 @@
       "title": "AnsibleModel",
       "type": "object"
     },
+    "AnsibleBuilderModel": {
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+        "workdir": {
+          "type": "string",
```

This should probably also have a title and a description.

```suggestion
          "title": "Workdir",
          "description": "Specify the path that contains ansible-builder manifest files"
          "type": "string",
```